.github/workflows/publish-npm.yml

```diff
@@ -15,4 +15,4 @@ jobs:
       package-dir: 'packages/snap'
       akeyless-api-gateway: https://api.gateway.akeyless.celo-networks-dev.org
       akeyless-github-access-id: p-kf9vjzruht6l
-      akeyless-token-path: /static-secrets/apps-tooling-circle/npm-publish-token
+      akeyless-token-path: /static-secrets/dev-tooling-circle/npm-publish-token
```

Comment:
to be honest i dont think this should be the same as our team name its not really just for us

Reply:
agreed, changed directory to NPM